src/components/ec-table-filter/ec-table-filter.vue

```diff
@@ -0,0 +1,167 @@
+<template>
+  <section
+    class="ec-table-filter"
+    data-test="ec-table-filter"
+  >
+    <component
+      :is="filter.component"
+      v-for="(filter, index) in filters"
+      :key="index"
+      :value="value[filter.name]"
+      v-bind="filter"
+      :label="filter.label"
+      :data-test="`ec-table-filter__filter-item-${index}`"
```

Comment:
Could you please add `ec-table-filter__filter-item` into this? 
```
:data-test="`ec-table-filter__filter-item ec-table-filter__filter-item-${index}`"
```

So then I can select all of them `findAllByDataTest('ec-table-filter__filter-item')` or only an individual one `findByDataTest('ec-table-filter__filter-item-1')`.